packages/core/src/stateUtils.ts

```diff
@@ -18,7 +18,9 @@ export const isLeafNode = (
 export function getChildren<TC, TE extends EventObject>(
   stateNode: StateNode<TC, any, TE>
 ): Array<StateNode<TC, any, TE>> {
-  return Object.keys(stateNode.states).map((key) => stateNode.states[key]);
+  return Object.keys(stateNode.states)
+    .map((key) => stateNode.states[key])
+    .filter((sn) => sn.type !== 'history');
```

Comment:
It's a pseudostate and thus it's never really in the configuration, so it should always be just ignored, and not treated as one. This matches now the [`getChildStates` procedure](https://www.w3.org/TR/scxml/#getChildStates) from SCXML